Frame.cc

```diff
@@ -11,7 +11,7 @@
 using namespace carta;
 using namespace std;
```

Comment:
Should we eliminate this if we're using `std::` explicitly? Are there other bare keywords from this namespace elsewhere in the file?